### ISSUE TITLE
Bug：The icon remains unchanged after moving the app

### ISSUE THREAD

**Issue by SuZhelevel6** [CocoCR300/flauncher#38]
Version：Release #3 December 2024 (2024.12.003)

Phenomenon: 
In App list （Grid）
When you long press the APP Card and then move it.
Actually, the app has been successfully moved, but the desktop icon has not changed

I Think it's come from `apps_grid.dart`  changes on commit 50913d13

Maybe you lose some code like

```
key: Key("${category.id}-${applications[index].packageName}"),
```

**Comment by SuZhelevel6** [comment]
By the way,
your modification in e4b0efda resulted that ：
When APP Card with Banner was focused， they lose the outer highlight ring 

Is this the effect you originally wanted?

**Comment by CocoCR300** [comment]
It took me a while to fix this because I was facing another issue after reverting the change you point out. But the bug just stopped happening without any other changes, the application was just crashing on me every time I tried to "reorder" an "app card". Guess it was something to do with some cache. Anyway, this is now fixed.

**Comment by SuZhelevel6** [comment]
fixed by 3f3482e2c9b2d8a616acc1527cf24ee0d3f10987
